### PR TITLE
Update example with deployment logging and consul support

### PR DESCRIPTION
**Pull request by pchojnacki**

### CODE REVIEW

File: gradle/pandora_common_deps.gradle [hunk 1]
```diff
@@ -5,6 +5,8 @@ dependencies {
     compile "io.dropwizard:dropwizard-core:$dropwizardVersion"
     compile "io.dropwizard:dropwizard-client:$dropwizardVersion"
 
+    compile 'com.wikia:dropwizard-logstash-encoder:1.0.1'
```
Comment (nmonterroso) [inline gradle/pandora_common_deps.gradle]: You should be able to change this to `1.0.2` now and uncomment the `port` setting in the `example.yaml.sample` file beflow

File: pandora-example/build.gradle [hunk 2]
```diff
@@ -1,8 +1,44 @@
...
+dependencies {
+    compile project(":pandora-core")
+}
+// end: deployment boilerplate
 
 mainClassName = "com.wikia.exampleservice.ExampleServiceApplication"
 
+// configure jvm parameters
+applicationDefaultJvmArgs = [
+        '-Ddw.server.applicationConnectors[0].port=\$PORT0', '-Ddw.server.adminConnectors[0].port=\$PORT1',
```
Comment (nmonterroso) [inline pandora-example/build.gradle]: How are `$PORT0` and `$PORT1` configured/defined?

Comment (pchojnacki) [inline pandora-example/build.gradle]: Marathon passes these to application.
But I'll add description of it to the example.

File: gradle/pandora_common_deps.gradle (hunk 1 above)
Comment (pchojnacki) [inline gradle/pandora_common_deps.gradle]: Cool I'll try it in a moment

**Comment by sqreek** [comment]
lgtm

**Comment by nmonterroso** [comment]
LGTM

**Comment by nandy-andy** [comment]
LGTM

**Pull request merged**